themes/theme-gmd/components/SnackBar/components/ActionButton/style.js

```diff
@@ -1,8 +1,11 @@
 import { css } from 'glamor';
+import color from 'color';
 import colors from 'Styles/colors';
 
+const textColorContrast = color(colors.accent).contrast(color('black'));
```

Comment:
we should check here the real background color from SnackBar/style. create a const for it

Reply:
Sounds reasonable. Will do